### DISCUSSION TITLE
[0.8] Next WeakAuras profiles

### DISCUSSION THREAD

**Discussion by MarioCatuogno** [MarioCatuogno/PanzaUI#59] [Polls]
Vote here for the next WAs for the current release ([0.8.0-RELEASE](https://github.com/MarioCatuogno/PanzaUI/releases/tag/0.8.0)).

**Comment by MarioCatuogno** [comment]
The plan should be the following:
- 0.8.1: Elemental Shaman
- 0.8.2: Holy Priest
- 0.8.3: Beast Mastery Hunter
- 0.8.4: Blood Death Knight

**Comment by maikelschippers** [comment]
Would be nice if tank specs were added.. im mainly playing tanks

**Reply by MarioCatuogno** [reply]
Hey @maikelschippers,
as of release 0.8.2, the following tank specs are supported:
- Brewmaster Monk
- Protection Paladin

I'm planning to add for release 0.8.4 the Blood Death Knight spec and in the future tank spec for Druid.

**Comment by maikelschippers** [comment]
No dh comming?
Groetjes,

Maikel & Marjon

Op 10 okt 2024 om 08:19 heeft Mario Catuogno ***@***.***> het volgende geschreven:

﻿

Hey @maikelschippers<https://github.com/maikelschippers>,
as of release 0.8.2, the following tank specs are supported:

  *   Brewmaster Monk
  *   Protection Paladin

I'm planning to add for release 0.8.4 the Blood Death Knight spec and in the future tank spec for Druid.

**Reply by MarioCatuogno** [reply]
Not in the near future, I'm not actually playing DH (and I don't have as much time as I would like to have).
The plan is to add a WA for every spec in the game.

**Comment by PandaRedd** [comment]
The prot warrior work in progress one that was briefly included was the
best sb / IP tracker that I've ever used. You are awesome, dude.

**Reply by MarioCatuogno** [reply]
Thanks for the kind words! 
I'm planning to add again the Protection Warrior WA in a complete form in the near future. Thinking of switching Blood Death Knight with Protection Warrior!

**Comment by MarioCatuogno** [comment]
Open a new discussion [here](https://github.com/MarioCatuogno/PanzaUI/discussions/64).
